crates/symbolicator/src/services/symbolication.rs

```diff
@@ -555,7 +555,7 @@ fn object_info_from_minidump_module_breakpad(ty: ObjectType, module: &CodeModule
 
     RawObjectInfo {
         ty,
-        code_id: Some(code_id),
+        code_id: Some(code_id.to_lowercase()),
```

Comment:
i wonder if it would be better to do a `CodeId::new(code_id).to_string()` version, but i guess this still doesn't matter much since this isn't master and should go away anyway.